packages/components/src/components/switch/switch.stories.ts

```diff
@@ -0,0 +1,174 @@
+import '../../solid-components';
+import { html } from 'lit-html';
+import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
+import { userEvent } from '@storybook/testing-library';
+import { waitUntil } from '@open-wc/testing-helpers';
+const { argTypes, parameters } = storybookDefaults('sd-switch');
+const { generateTemplate } = storybookTemplate('sd-switch');
+const { overrideArgs } = storybookHelpers('sd-switch');
+
+export default {
+  title: 'Components/sd-switch',
+  component: 'sd-switch',
+  args: overrideArgs([{ type: 'slot', name: 'default', value: 'Default Slot' }]),
+  argTypes,
+  parameters: {
+    ...parameters,
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/file/sFzAnWIy3ck28dF4vv1RnE/Switch?node-id=642%3A1032&mode=dev'
+    }
+  }
+};
+
+/**
+ * Default: This shows sd-switch in its default state.
+ */
+
+export const Default = {
+  render: (args: any) => {
+    return generateTemplate({ args });
+  }
+};
+
+/**
+ * Use the disabled attribute to disable an input checkbox. Clicks will be suppressed until the disabled state is removed
```

Comment:
I don't know if I would go with the naming checkbox 🤔 I would Propose to take the name from the component -> switch